### ISSUE TITLE
MenuButton's Font Icon hover color

### ISSUE THREAD

**Issue by robbiezl** [mkpaz/atlantafx#100]
The Icon color of FontIcon in Contextmenu is correct，but the Icon color of FontIcon in MenuButton is not corrent.
I found in the CSS,the menu-button font icon fill color is always -color-button-fg.

But in contextmenu the menu-item's icon color sometimes  -color-fg-emphasis or -color-button-fg

**Comment by mkpaz** [comment]
Good catch. I never checked how font icons look in the menu button popup. That should be tested.
